app/services/current-session.js

```diff
@@ -44,6 +45,11 @@ export default class CurrentSessionService extends Service {
   }
 
   may(permission, checkImpersonator = false) {
+    if (this.userAgent.browser.isMobile || this.userAgent.browser.isEdge) {
```

Comment:
checked with smartphone.
`this.userAgent.browser.isMobile` does not seem to be a thing on samsung.
Is this phone specific?